src/test/java/com/salesforce/dynamodbv2/testsupport/DefaultTestSetup.java

```diff
@@ -185,7 +186,7 @@ protected MtAmazonDynamoDbContextProvider getMtContext() {
         CreateTableRequestBuilder baseBuilder = CreateTableRequestBuilder.builder()
                 .withTableName(TABLE1)
                 .withAttributeDefinitions(new AttributeDefinition(HASH_KEY_FIELD, hashKeyAttrType))
-                .withKeySchema(new KeySchemaElement(HASH_KEY_FIELD, KeyType.HASH))
+                .withKeySchema(getKeySchema(HASH_KEY_FIELD, HASH))
```

Comment:
Most changes in this file are just refactoring and don't change logic. The only logical change is that in TABLE5, I'm adding 2 new GSIs that share a common non-PK field, so I can write the test on updating a field used in multiple GSIs.